<launch>
    <arg name="launch_realsense" default="true"/>
    <group if="$(arg launch_realsense)">
        <include file="$(find realsense2_camera)/launch/rs_rgbd.launch">
            <arg name="enable_pointcloud" value="false"/>
        </include>
    </group>
    <node name="orb_slam2_rgbd" pkg="orb_slam2_ros" type="orb_slam2_rgbd"
          output="screen">
        <param name="voc_file" value="$(env ORBSLAM2_LIBRARY_PATH)/Vocabulary/ORBvoc.bin"/>
        <param name="settings_file" value="$(find orb_slam2_ros)/cfg/realsense_d435.yaml"/>
        <param name="rgb_topic" value="/camera/color/image_raw"/>
        <param name="depth_topic" value="/camera/aligned_depth_to_color/image_raw"/>
    </node>
</launch>